src/app/_components/Form/FormCheckbox.tsx

```diff
@@ -1,26 +1,57 @@
 import { Checkbox, type CheckboxProps, Field, Label } from '@headlessui/react'
 import { Check } from '@phosphor-icons/react/dist/ssr'
+import { clsx } from 'clsx'
 
 import { Icon } from '@/components/Icon'
 
 type ExcludedHeadlessUIProps = 'className'
 
 export type FormCheckboxProps = {
   label: string
+  labelSize?: 'sm' | 'md'
 } & Omit<CheckboxProps, ExcludedHeadlessUIProps>
 
-export function FormCheckbox({ label, ...rest }: FormCheckboxProps) {
+const TOUCH_TARGET = {
+  base: 'size-5',
+  clickArea: 'p-3.5',
+  clickAreaOffset: '-m-3.5',
+}
+
+export function FormCheckbox({
+  label,
+  labelSize = 'md',
+  ...rest
+}: FormCheckboxProps) {
   return (
     <Field className="flex items-center gap-3">
       <Checkbox
         {...rest}
-        className="group size-5 rounded bg-brand-100 p-0.5 text-brand-100 focus:brand-outline data-[disabled]:cursor-not-allowed data-[checked]:bg-brand-400"
+        className={clsx(
+          'group cursor-pointer focus:outline-none',
```

Comment:
Let's work on the focus styles.

Reply:
We should have a medium blue focus style for white background elements.

Reply:
✅